robolectric/src/test/java/org/mockito/configuration/MockitoConfiguration.java

```diff
@@ -0,0 +1,9 @@
+package org.mockito.configuration;
+
+public class MockitoConfiguration extends DefaultMockitoConfiguration {
+
+    @Override
+    public boolean enableClassCache() {
+        return false;
```

Comment:
instead of returning false. can we do something like

`return !isMultiApiRobolectricTestRunner`